myTrip/photo/models.py

```diff
@@ -1,25 +1,35 @@
 """Module contain photo model class and methods."""
 
+from datetime import datetime
+
 from django.core.exceptions import ObjectDoesNotExist
 from django.db import models
 
+from checkpoint.models import Checkpoint
+from registration.models import CustomUser
+from trip.models import Trip
+
 
 class Photo(models.Model):
     """
     Photo
     :argument id: int - auto generated primary key
     :argument src: url - photo source link
-    :argument user_id: int - ToDo foreign key to User model id
-    :argument trip_id: int - Todo foreign key to Trip model id
-    :argument checkpoint_id: int - ToDo foreign to Checkpoint model id
-    :argument description: str - description to photo.
+    :argument user: - foreign key to User model
+    :argument trip: - foreign key to Trip model
+    :argument checkpoint: - foreign to Checkpoint model
+    :argument description: str - description to photo
+    :argument created_at: date - time when created
+    :argument updated_at: date - time when updated.
     """
 
     src = models.URLField(max_length=200)
```

Comment:
Max_length is used by default of 200, should be described if the value is different from the default.
Https://docs.djangoproject.com/en/1.11/ref/models/fields/#urlfield